include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp

```diff
@@ -0,0 +1,18 @@
+namespace aikido {
+namespace statespace {
+namespace dart {
+
+MetaSkeletonStateSpaceSaver::MetaSkeletonStateSpaceSaver(MetaSkeletonStateSpacePtr _space)
+: mSpace(std::move(_space))
+, mPositions(mSpace->getMetaSkeleton()->getPositions())
+{
+}
+
+MetaSkeletonStateSpaceSaver::~MetaSkeletonStateSpaceSaver()
+{
+  mSpace->getMetaSkeleton()->setPositions(mPositions);
```

Comment:
Nit: It would be nice if we have a simple validity check whether the size of `mPositions` is still equal to the dofs of the `MetaSkeleton` (if not, we print a warning). I understand it wouldn't be able to catch other structural changes such as changing the joint type with the same dofs, but at least we could catch dimension changing cases.